test/cli-test.ts

```diff
@@ -638,6 +627,20 @@ parallel(
 );
 
 describe('Spellchecker CLI (not parallel)', () => {
+  it('exits with an error when no arguments are provided', async () => {
+    // Remove .spellcheckerrc.yml so that Spellchecker CLI doesn't use it in this test.
+    const config = readFileSync('.spellcheckerrc.yml', { encoding: 'utf8' });
+    rmSync('.spellcheckerrc.yml');
```

Comment:
This test fails without these lines, proving that Spellchecker CLI is still reading from `.spellcheckerrc.yml`.